packages/actions/index.ts

```diff
@@ -282,28 +307,6 @@ export const getTwoPartyBalance = (state) => async(user: User, friend: Friend) =
   return new Balance({ relativeToNickname: friend.nickname, relativeTo: friend.address, amount: amount })
 }
 
-//Needs a selector
-export const getAccountInformation = () => {
-  return async (dispatch, getState) => {
-    let { address, nickname, email } = getUser(getState())()
-
-    if (nickname.length === 0 || email.length === 0) {
-      try {
-        nickname = await creditProtocol.getNickname(address)
-        email = await creditProtocol.getEmail(address)
-      } catch (e) {}
-    }
-
-    const accountInformation: { nickname?: string, email?: string, balance?: number } = { nickname, email }
-    try {
-      accountInformation.balance = await creditProtocol.getBalance(address, getPrimaryCurrency(getState()))
```

Comment:
The call to `getBalance` was removed, was this intentional?

Reply:
yes, that was not being used anymore